### PR TITLE
Melpa prep

### PR DESCRIPTION
**Pull request by rnikoopour**

### CODE REVIEW

File: lsp-docker.el [hunk 1]
```diff
@@ -3,7 +3,11 @@
 ;; Copyright (C) 2019  Ivan Yonchovski
 
 ;; Author: Ivan Yonchovski <yyoncho@gmail.com>
-;; Keywords:
+;; URL: https://github.com/emacs-lsp/lsp-docker
+;; Keywords: languages langserver
+;; Version: 1.0.0
+;; Package-Requires: ((emacs "25.1") (dash "2.14.1"))
```
Comment (yyoncho) [inline lsp-docker.el]: dont we need lsp-mode as well?

Comment (rnikoopour) [reply lsp-docker.el]: I overlooked it.  Will update shortly.

File: lsp-docker.el [hunk 2]
```diff
@@ -20,21 +24,23 @@
 
 ;;; Commentary:
 
-;;
+;; Runs lanaguage servers in containers
```
Comment (yyoncho) [inline lsp-docker.el]: typo: language

File: README.org [hunk 3]
```diff
@@ -94,12 +94,33 @@ Docker image + scripts for running =lsp-mode= in =docker= environment.
...
+       (list :server-id 'ts-ls :docker-server-id 'tsls-docker :server-command "typescript-language-server --stdio")
+       (list :server-id 'tfls :docker-server-id 'tfls-docker :server-command "terraform-lsp")))
+
+    (require 'lsp-docker)
+    (lsp-docker-init-clients
       :path-mappings '(("path-to-projects-you-want-to-use" . "/projects"))
       :docker-image-id "yyoncho/lsp-emacs-docker"
       :docker-container-name "lsp-container"
+      :client-packages: lsp-docker-client-packages
+      :client-configs: lsp-docker-client-configs
```
Comment (yyoncho) [inline README.org]: redundant `:`?

**Comment by yyoncho** [comment]
Thank you!

**Pull request merged**